backend/src/graphql/resolver/TransactionResolver.ts

```diff
@@ -289,64 +228,90 @@ export class TransactionResolver {
   ): Promise<TransactionList> {
     // load user
     const userRepository = getCustomRepository(UserRepository)
-    let userEntity: dbUser | undefined
-    if (userId) {
-      userEntity = await userRepository.findOneOrFail({ id: userId }, { withDeleted: true })
-    } else {
-      userEntity = await userRepository.findByPubkeyHex(context.pubKey)
+    const user = userId
+      ? await userRepository.findOneOrFail({ id: userId }, { withDeleted: true })
+      : await userRepository.findByPubkeyHex(context.pubKey)
+    let limit = pageSize
+    let offset = 0
+    let skipFirstTransaction = false
+    if (currentPage > 1) {
+      offset = (currentPage - 1) * pageSize - 1
```

Comment:
```suggestion
      offset = (currentPage - 1) * pageSize
```
I want to skip the whole page or is it on purpose to see the last transaction of the first page as the first transaction of the second page?

Reply:
I have no clue, all I did is remove the function encapsulating this. I figured that I'd like to solve this puzzle sometimes else and make this whole section more readable and well defined - if its ok I would not do that in this PR